.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,171 @@
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: |
+        To avoid invalid reports, please check and confirm following checkboxes.
+      options:
+        - label: This site DOES NOT contains sexually explicit material;
+          required: true
+        - label: DNS filters were updated before redroduced an issue;
+          required: false
+        - label: AdGuard product version is up-to-date;
+          required: false
+        - label: Browser version is up-to-date;
+          required: false
+        - label: If the site or app is broken, disabling AdGuard protection resolves an issue.
+          required: true
+
+  - type: dropdown
+    id: adguard-dns
+    attributes:
+      label: What DNS-related product do you use?
+      multiple: true
+      options:
+        - AdGuard public DNS
+        - AdGuard private DNS
+        - AdGuard Home
+    validations:
+      required: true
+
+  - type: dropdown
+    id: adguards
+    attributes:
+      label: What ad blocker do you use?
+      description: If you use an ad blocker, please specify its version in the next step
+      multiple: true
+      options:
+        - DNS only
+        - AdGuard Browser Extension
+        - AdGuard Content Blocker
+        - AdGuard for Android
+        - AdGuard for iOS
+        - AdGuard for Mac
+        - AdGuard for Safari
+        - AdGuard for Windows
+        - Other ad blocker
+    validations:
+      required: true
+
+  - type: input
+    id: version
+    attributes:
+      label: AdGuard version
+      description: What version of our software are you running?
+      placeholder: ex. 7.7
+    validations:
+      required: false
+
+  - type: dropdown
+    id: what-happened
+    attributes:
+      label: What type of problem have you encountered?
+      description: If the problem does not fall under any category that is listed here, please contact our tech support - support@adguard.com
+      multiple: true
+      options:
+        - Missed ads
+        - Missed analytics or tracker
+        - Website or app doesn't work properly
+    validations:
+      required: true
+
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browser is used?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Opera
+        - Yandex Browser
+        - Other(specify in comment below)
+        - A problem is with an app or a device
+    validations:
+      required: true
+
+  - type: dropdown
+    id: browsers_type
+    attributes:
+      label: Which device is used?
+      multiple: false
+      options:
+        - Desktop
+        - Mobile
+        - Other device
+    validations:
+      required: true
+
+  - type: input
+    id: url
+    attributes:
+      label: Where did you encounter the problem?
+      description: Please enter address of the page or download link of the affected app.
```

Please enter the page address 
```suggestion
      description: Please enter the page address or a download link of the affected app.
```